php/class-fieldmanager-field.php

```diff
@@ -40,217 +46,257 @@
+	 * are always saved).
+	 *
 	 * @var boolean
-	 * If true, save empty elements to DB (if $this->limit != 1; single elements are always saved)
 	 */
-	public $save_empty = False;
+	public $save_empty = false;
 
 	/**
-	 * @var boolean
 	 * Do not save this field (useful for fields which handle saving their own data)
+	 *
+	 * @var boolean
 	 */
-	public $skip_save = False;
+	public $skip_save = false;
 
 	/**
-	 * Save this field additionally to an index
+	 * Save this field additionally to an index.
+	 *
 	 * @var boolean
 	 */
-	public $index = False;
+	public $index = false;
 
 	/**
 	 * Save the fields to their own keys (only works in some contexts). Default
 	 * is true.
+	 *
 	 * @var boolean
 	 */
 	public $serialize_data = true;
 
 	/**
+	 * Optionally generate field from datasource. Used by Fieldmanager_Autocomplete
+	 * and Fieldmanager_Options.
+	 *
 	 * @var Fieldmanager_Datasource
-	 * Optionally generate field from datasource. Used by Fieldmanager_Autocomplete and Fieldmanager_Options.
 	 */
-	public $datasource = Null;
+	public $datasource = null;
 
 	/**
-	 * @var array[]
 	 * Field name and value on which to display element. Sample:
 	 * $element->display_if = array(
-	 *	'src' => 'display-if-src-element',
-	 *	'value' => 'display-if-src-value'
+	 *     'src' => 'display-if-src-element',
+	 *     'value' => 'display-if-src-value',
 	 * );
+	 *
+	 * @var array[]
 	 */
 	public $display_if = array();
 
 	/**
-	* @var string
-	* Where the new item should to added ( top/bottom ) of the stack. Used by Add Another button
-	* "top|bottom"
-	*/
-	public $add_more_position = "bottom";
+	 *
+	 * Where the new item should to added ( top/bottom ) of the stack. Used by Add Another button
+	 * "top|bottom".
+	 *
+	 * @var string
+	 */
+	public $add_more_position = 'bottom';
 
 	/**
+	 * If true, remove any default meta boxes that are overridden by Fieldmanager fields.
+	 *
 	 * @var boolean
-	 * If true, remove any default meta boxes that are overridden by Fieldmanager fields
 	 */
-	public $remove_default_meta_boxes = False;
+	public $remove_default_meta_boxes = false;
 
 	/**
+	 * The path to the field template.
+	 *
 	 * @var string Template
-	 * The path to the field template
 	 */
-	public $template = Null;
+	public $template = null;
 
 	/**
+	 * If $remove_default_meta_boxes is true, this array will be populated with
+	 * the list of default meta boxes to remove.
+	 *
 	 * @var array
-	 * If $remove_default_meta_boxes is true, this array will be populated with the list of default meta boxes to remove
 	 */
 	public $meta_boxes_to_remove = array();
 
 	/**
+	 * The default value for the field, if unset.
+	 *
 	 * @var mixed Default value
-	 * The default value for the field, if unset
 	 */
 	public $default_value = null;
 
 	/**
-	 * @var callable|null
 	 * Function that parses an index value and returns an optionally modified value.
+	 *
+	 * @var callable|null
```

Comment:
There are a few `@var`s here in the form of `type|null`, but in the changes to `Fieldmanager_Autocomplete` above, there are properties initialized as `null` but documented as just `type`. Not sure whether that deserves standardization.